resolver/resolver.go

```diff
@@ -15,10 +19,33 @@ var (
 	ErrNotSupported = errors.New("unsupported")
 )
 
+// IncludePath is the directory where make includes reside and to which mmake
+// will download remote includes for the duration of a particular execution
+// of mmake.
+var IncludePath = DefaultIncludePath
+
 // Interface for resolving and fetching includes.
 type Interface interface {
 	// Get must return the contents of the file,
 	// ErrNotFound when the file is not found,
 	// or an error.
 	Get(path string) (io.ReadCloser, error)
 }
+
+// GetIncludePath allows the include path to be overridden.
+func GetIncludePath(args []string) string {
+	// always reset the global IncludePath to the default
+	IncludePath = DefaultIncludePath
```

Comment:
is this line necessary if it's defaulted above?

Reply:
That's a good question. I know I had a reason for doing it this way, but that reason escapes me at this moment. I'll have to take a closer look at the whole diff when I get back to a computer.

Reply:
I think I added this line so the test results would be consistent. Without it, the expected outcome for later tests would not match. This being the case, I'll just reset `IncludePath` in the test instead.